notebooks/14_regression/index.md

```diff
@@ -0,0 +1,222 @@
+# Regression
+
+# Table of Contents
+
+- [Regression](#regression)
+- [Table of Contents](#table-of-contents)
+- [Introduction](#introduction)
+- [1 - Linear Regression](#2---linear-regression)
+  - [Loss Function](#loss-function)
+    - [Mean Squared Error (MSE)](#mean-squared-error-mse)
+  - [Finding $\hat{w}$](#finding-hatw)
+    - [Gradient Descent](#gradient-descent)
+    - [Normal Equation](#normal-equation)
+- [2 - Learning Curves Using Polynomials](#3---learning-curves-using-polynomials)
+  - [Reduction to Linear Regression](#reduction-to-linear-regression)
+  - [Overfitting](#overfitting)
+    - [Using Validation Set (Held-Out Data)](#using-validation-set-held-out-data)
+    - [Regularization](#regularization)
+- [3 - Logistic Regression](#5---logistic-regression)
+  - [How to calculate probabilities](#how-to-calculate-probabilities)
+  - [defining line _l_ and cost function $J(\theta)$](#defining-line-l-and-cost-function-jtheta)
```

Comment:
Use the same format in headers. If you start all your words except prepositions, with a capital case, do the same for other headers.